include/dem/particle_particle_broad_search.h

```diff
@@ -81,6 +78,24 @@ class ParticleParticleBroadSearch
       dim>::particle_particle_candidates &local_contact_pair_candidates,
     typename dem_data_containers::dem_data_structures<
       dim>::particle_particle_candidates &ghost_contact_pair_candidates);
+
+  /**
+   * Stores the candidate particle-particle collision pairs with a given
+   * particle iterator.
```

Comment:
You explain what the function does, but can you explain a bit what is is used for? Just give an example